build.sbt

```diff
@@ -150,6 +152,21 @@ lazy val commonSettings = Seq(
   )
 ) ++ compilerPlugins
 
+//Settings
+lazy val clientSettings = Seq(
+  libraryDependencies ++= Seq(
+    "com.pepegar" %% "hammock-core" % V.hammock,
+    "com.pepegar" %% "hammock-apache-http" % V.hammock,
```

Comment:
I believe it's better to user `hammock-asynchttpclient`, which is asynchronous and more common.